packages/cli-repl/package.json

```diff
@@ -46,6 +46,7 @@
     "analytics-node": "^3.4.0-beta.1",
     "ansi-escape-sequences": "^5.1.2",
     "askpassword": "^1.2.1",
+    "chai-as-promised": "^7.1.1",
```

Comment:
This should probably be in `devDependencies`? :)

Reply:
Ooops, right, will fix!